lang/haw.js

```diff
@@ -210,10 +210,10 @@ export default {
 	"components.tag-list.interactive-label": "Papa inoa inoa, {count} mau mea",
 	"components.tag-list.num-hidden": "+ {count} hou aku",
 	"components.tag-list.role-description":
-		`{count, plural,
-			=0 {Tag List with 0 items}
-			one {Tag List with {count} item}
-			other {Tag List with {count} items}
+		`{helu, nui, 
```

Comment:
This looks to have been translated incorrectly. I'll fix up the formatting and then submit an issue about it.